src/sendCommercialMetrics.ts

```diff
@@ -1,72 +1,186 @@
 import { log } from '@guardian/libs';
 import { EventTimer } from './EventTimer';
 
-type Metrics = {
+type Metric = {
 	name: string;
 	value: number;
 };
 
-type Properties = {
+type Property = {
 	name: string;
-	value: string;
+	value: string | number;
```

Comment:
I’d be tempted to keep this a `string` as that’s what the backend expects.

Reply:
Yes, the value is a string either way so this is an unnecessary alternative 👍🏻